docs/orchestrating-elastic-stack-applications/agent-fleet.asciidoc

```diff
@@ -484,6 +486,159 @@ spec:
 Please note that the environment variables related to policy selection mentioned in the Elastic Agent link:https://www.elastic.co/guide/en/fleet/current/agent-environment-variables.html[docs] like `FLEET_SERVER_POLICY_ID` will be managed by the ECK operator.
 
 
+[id="{p}-elastic-agent-running-as-a-non-root-user"]
+// tag::configuration-example-elastic-agent-running-as-a-non-root-user[]
+=== Running as a non-root user
+
+In order to run Elastic Agent as a non-root user you must choose how you want to persist data to the Agent's volume.
+
+1. Run Elastic Agent with an `emptyDir` volume. This has the downside of not persisting data between restarts of the Elastic Agent which can duplicate work done by the previous running Agent.
+2. Run Elastic Agent with a `hostPath` volume in addition to a `Daemonset` running as `root` that maintains permissions for the `agent` user.
+
+In addition to these decisions, if you are running Elastic Agent in Fleet mode as a non-root user you must configure Kibana to accept the CA of the Elasticsearch Cluster.
```

Comment:
```suggestion
In addition to these decisions, if you are running Elastic Agent in Fleet mode as a non-root user, you must configure `certificate_authorities.ssl` in each `xpack.fleet.outputs` to trust the CA of the Elasticsearch Cluster.
```
I think this sentence can be misleading. IIUC the CA must be configured in the fleet outputs configuration which is itself in the Kibana resource:

```yaml
apiVersion: kibana.k8s.elastic.co/v1
kind: Kibana
metadata:
  name: kibana
spec:
  version: 8.7.0
  count: 1
  elasticsearchRef:
    name: elasticsearch
  config:
    xpack.fleet.agents.fleet_server.hosts: ["https://fleet-server-agent-http.default.svc:8220"]
    xpack.fleet.outputs:
    - id: eck-fleet-agent-output-elasticsearch
      is_default: true
      name: eck-elasticsearch
      type: elasticsearch
      hosts: ["https://elasticsearch-es-http.default.svc:9200"]
      ssl:
        certificate_authorities: ["/mnt/elastic-internal/elasticsearch-association/default/elasticsearch/certs/ca.crt"]
```

But there is no need to change of set `elasticsearch.ssl.certificateAuthorities`.